src/help.txt

```diff
@@ -0,0 +1,10 @@
+usage: jotdown [option] <file> ...
+
+arguments:
+    file            one or more djot source file(s)
+
+options:
+    -h --help       show this text
+    -v --version    show the version number
+    -f --format     the output format {html|latex}
```

Comment:
we don't support any formats except for html yet, so I don't think we need this flag yet

Reply:
True, I seem to have forgotten to update the help text after drafting it! I'll leave "format" in the implementation, but remove it from the help.

Reply:
I would remove it from the implementation also, I think doc and impl should match.